src/extension.ts

```diff
@@ -301,32 +291,55 @@ function configureRunTestFromCodeLens() {
   vscode.commands.registerCommand(Commands.RUN_TEST_FROM_CODELENS, runFromCodeLens);
 }
 
-export function activate(context: ExtensionContext): void {
-  testElixir();
-  detectConflictingExtension("mjmcloug.vscode-elixir");
-  // https://github.com/elixir-lsp/vscode-elixir-ls/issues/34
-  detectConflictingExtension("sammkj.vscode-elixir-formatter");
-
-  configureRunTestFromCodeLens()
-  configureCopyDebugInfo(context);
-  configureExpandMacro(context);
-  configureDebugger(context);
-  configureTerminalLinkProvider(context);
-
+function startClient(context: ExtensionContext, clientOptions: LanguageClientOptions): LanguageClient {
   const command =
     os.platform() == "win32" ? "language_server.bat" : "language_server.sh";
 
   const serverOpts = {
     command: context.asAbsolutePath("./elixir-ls-release/" + command),
   };
 
-  // If the extension is launched in debug mode then the debug server options are used
-  // Otherwise the run options are used
```

Comment:
Did you mean to remove this comment? I think it is valuable enough to keep.

Reply:
I'll change it as original one was incorrect - regardless of the mode we pass the same options

Reply:
hmm, I would consider the original message technically correct but perhaps a little misleading. A more clear wording would be good.